python/mxnet/gluon/contrib/estimator/batch_processor.py

```diff
@@ -99,7 +100,9 @@ def fit_batch(self, estimator,
             pred = [estimator.net(x) for x in data]
             loss = [estimator.loss(y_hat, y) for y_hat, y in zip(pred, label)]
 
-        for l in loss:
-            l.backward()
+            for l in loss:
+                l.backward()
```

Comment:
You changed the indent

```suggestion
        for l in loss:
            l.backward()
```